crates/aptos-crypto/src/poseidon_bn254/alt_fr.rs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use ark_ff::{BigInteger, PrimeField as ArkPrimeField};
+use ff::PrimeField;
+
+/// Unfortunately, `neptune` is not compatible with the arkworks ff traits.
+#[derive(PrimeField)]
+#[PrimeFieldModulus = "21888242871839275222246405745257275088548364400416034343698204186575808495617"]
+#[PrimeFieldGenerator = "5"]
+#[PrimeFieldReprEndianness = "little"]
+pub struct AltFr([u64; 4]);
```

Comment:
can you just use use num_bigint::BigUint or something?

Reply:
Unlikely. The Poseidon `neptune` library. requires certain traits to be implemented.